src/store/saga/rootSaga.js

```diff
@@ -2,13 +2,31 @@ import { all } from 'redux-saga/effects';
 import {
   watchUser,
   watchLoginUser,
-  watchLogoutUser
+  watchLogoutUser,
+  watchCreateUser,
+  watchUpdateUser
 } from './userSagas';
+import {
+  watchGetTasksList,
+  watchUpdateTask,
+  watchDeleteTask,
+  watchGetOneTask,
+  watchCreateTask,
+  watchUpdateCurrentTask
+} from './tasksSagas';
 
 export function* rootSaga() {
   yield all([
     watchUser(),
     watchLoginUser(),
-    watchLogoutUser()
+    watchLogoutUser(),
+    watchCreateUser(),
+    watchUpdateUser(),
+    watchGetTasksList(),
+    watchUpdateTask(),
+    watchDeleteTask(),
+    watchGetOneTask(),
+    watchCreateTask(),
+    watchUpdateCurrentTask()
```

Comment:
it is better combine all sagas in one
```
export function* watchAllTask() {
  yield all([
    takeEvery(GET_TASKS_LIST_ASYNC, getTasksList),
    takeEvery(UPDATE_TASK_ASYNC, update),
   ...
  ]);
}
```

and then use in rootSaga one function
```
export function* rootSaga() {
   yield all([watchAllTask(), ...] 
}
```